multivac/sensors/test_status.py

```diff
@@ -91,6 +106,11 @@ def test_status_iter(log_fh, cache_filepath=None):
                 yield res
                 continue
 
+    # if there are tests with no result, save them as failed.
+    for wid in saved_data:
+        res = saved_data.get(wid)  # get tuple (test name, test conf)
+        yield (res[0], res[1], 'failed')
```

Comment:
1. Are you sure that it is `failed` and not `fail`? Test run print `[ fail ]` about test;
2. `failed` - magic string. Can you put this into any constant? I'm not sure how to do it better.

Reply:
fixed